src/components/SiteDetail.js

```diff
@@ -92,48 +72,67 @@ const SiteDetail = ({ selectSite }) => {
   );
 
   const SiteDetailCards = protocolsArray.map(protocol => {
-    const loadedSiteProtocol = loadedSite && loadedSite.properties.protocols[protocol.name];
+    const bleachingProtocol = protocol.name === 'bleachingqc';
+    const transectProperty = bleachingProtocol ? protocol.property : protocol.name;
+    const bleachingSubItems = loadedSite && loadedSite.properties.protocols['colonies_bleached'];
+    const loadedSiteProtocol = loadedSite && loadedSite.properties.protocols[transectProperty];
+
     const dataPolicy = loadedSiteProtocol && loadedSite.properties[`data_policy_${protocol.name}`];
     const setToPrivate = dataPolicy === 'private';
 
-    const generatePrivateLabel = protocol => {
-      const protocolName =
-        protocol === 'beltfish' ? 'Fish Belt' : 'Benthic: PIT, LIT and Habitat Complexity';
-      return `This data is unavailable because ${protocolName} Sample Units are set to Private for this project.`;
-    };
+    const generatePrivateLabel = protocol_title =>
+      `This data is unavailable because ${protocol_title} Sample Units are set to Private for this project.`;
 
-    const convertContent = (content, protocol) => {
+    const convertBleachingContent = content => {
       return (
         content &&
-        content.map(item => {
-          const attribute = Object.keys(item)[0];
-          const value =
-            protocol === 'beltfish' ? Object.values(item)[0] : Object.values(item)[0] * 100;
-          return { x: attribute, y: value };
+        bleachingCategories.map(item => {
+          return { x: item.name, y: content[item.type] };
```

Comment:
This one is tricky, because we generally prefer pure functions over impure ones. Are you able to make sure everything that is computed in this function comes from an argument to this function? Higher-order-functions are fine if you need to use them.